### PR TITLE
Changing the type for expiry_timestamp back to a number 

### LINKED ISSUE #5080: REST API /tokens/{id} response expiry_timestamp incompatible type change
### Description

`/tokens/{id}` response `expiry_timestamp` type is accidentally changed from number to string in PR #4914, this breaks existing customers consuming the data.

This also exposes a bug that the actual type in the response mismatches what's defined in our our openapi spec prior to 0.70.0.

We need to change the type `expiry_timestamp` back to a number, and fix the openapi spec accordingly.

### Steps to reproduce

Compare non-null `expiry_timestamp` of a token prior to and at release 0.70.0

### Additional context

_No response_

### Hedera network

testnet

### Version

v0.70.0

### Operating system

None

### PR DESCRIPTION
**Pull request by mgoelswirlds**
**Description**:

Fixing accidental changes from the  PR https://github.com/hashgraph/hedera-mirror-node/pull/4914
Fixing a bug that the actual type in the response mismatches what's defined in our our openapi spec prior to 0.70.0.

This PR modifies 
* Changes the type expiry_timestamp back to a number
* Fixes the openapi spec to change the return type to a number for expiry_timestamp.

**Related issue(s)**:
Fixes #5080

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-mirror-rest/__tests__/specs/tokens/{id}/no-params.json [hunk 1]
```diff
@@ -71,7 +71,7 @@
     "created_timestamp": "1234567890.000000002",
     "decimals": "1000",
     "deleted": null,
-    "expiry_timestamp": "1241567889.000000002",
+    "expiry_timestamp": 1241567889000000002,
```
Comment (steven-sheehy) [inline hedera-mirror-rest/__tests__/specs/tokens/{id}/no-params.json:74]: Let's also update another spec to have a non-calculated `expiry_timestamp`

**Review by xin-hedera — APPROVED** [review]
LGTM

File: hedera-mirror-rest/api/v1/openapi.yml [hunk 2]
```diff
@@ -2928,7 +2928,10 @@ components:
           example: true
           nullable: true
         expiry_timestamp:
-          $ref: '#/components/schemas/TimestampNullable'
+          example: null
```
Comment (steven-sheehy) [inline hedera-mirror-rest/api/v1/openapi.yml:2931]: Should provide a valid populated example

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

**Review by steven-sheehy — APPROVED** [review]
LGTM

**Review by xin-hedera — APPROVED** [review]

**Pull request merged**